.gitignore

```diff
@@ -23,3 +23,5 @@
 /sail-cheri-mips
 
 /cheri_concentrate_listings/cheri-cap-lib
+
+/.DS_Store
```

Comment:
(Copying from https://github.com/CTSRD-CHERI/cheri-specification/commit/d659b2902cbbf2cd7e523ed86552a2551cb2d835#r123866209 as it doesn't show here)

This belongs in your system's global .gitignore. Repositories' .gitignore files are for things specific to the repository.

Reply:
dropped this change